ansible/roles/test/templates/pfc_storm_stop_arista.j2

```diff
@@ -1,5 +1,5 @@
 bash
 cd /mnt/flash
-sudo pkill -f {{pfc_gen_file}}
+sudo pkill -f "sudo python {{pfc_gen_file}} -p {{(1).__lshift__(pfc_queue_index)}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface | replace("Ethernet", "et") | replace("/", "_")}} -r {{ansible_eth0_ipv4_addr}}"
```

Comment:
Do we need to change pfc_storm_stop_mlnx.j2 as well? Is pfc storm stop per-port, per-queue granularity used in this PR?

Reply:
Yes, please help us change pfc_storm_stop_mlnx.j2 and create PR. Yes, it is used in the multiple port storming case in this PR.